Packs/AWS-IAMIdentityCenter/Integrations/AWSIAMIdentityCenter/AWSIAMIdentityCenter.yml

```diff
@@ -180,36 +621,433 @@ script:
+    - contextPath: AWS.IAMIdentityCenter.User.Title
+      description: Title of the user.
+      type: String
+    - contextPath: AWS.IAMIdentityCenter.User.PreferredLanguage
+      description: Preferred language of the user.
+      type: String
+    - contextPath: AWS.IAMIdentityCenter.User.Locale
+      description: Geographical region or location of the user.
+      type: String
+    - contextPath: AWS.IAMIdentityCenter.User.Timezone
+      description: Time zone of the user.
+      type: String
+    - contextPath: AWS.IAMIdentityCenter.User.IdentityStoreId
+      description: Globally unique identifier for the identity store.
+      type: String
+    name: aws-iam-identitycenter-get-user-by-email
+  - arguments:
+    - auto: PREDEFINED
+      description: The AWS Region, if not specified the default region will be used.
+      name: region
+      predefined:
+      - us-east-1
+      - us-east-2
+      - us-west-1
+      - us-west-2
+      - ca-central-1
+      - eu-west-1
+      - eu-central-1
+      - eu-west-2
+      - ap-northeast-1
+      - ap-northeast-2
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+    - description: Role Arn.
+      name: roleArn
+      required: false
+    - description: Role Session Duration.
+      name: roleSessionDuration
+      required: false
+    - description: Role Session Name.
+      name: roleSessionName
+      required: false 
+    - description: Identity Store ID.
+      name: IdentityStoreId
+      required: false
+    - description: The name of the user to remove.
+      name: userName
+      required: true
+    description: Removes the entered user.
+    name: aws-iam-identitycenter-delete-user
+  - arguments:
+    - auto: PREDEFINED
+      description: The AWS Region, if not specified the default region will be used.
+      name: region
+      predefined:
+      - us-east-1
+      - us-east-2
+      - us-west-1
+      - us-west-2
+      - ca-central-1
+      - eu-west-1
+      - eu-central-1
+      - eu-west-2
+      - ap-northeast-1
+      - ap-northeast-2
+      - ap-southeast-1
+      - ap-southeast-2
+      - ap-south-1
+      - sa-east-1
+      - eu-north-1
+      - eu-west-3
+      required: false
+    - description: Role Arn.
+      name: roleArn
+      required: false
+    - description: Role Session Duration.
+      name: roleSessionDuration
+      required: false
+    - description: Role Session Name.
+      name: roleSessionName
+      required: false 
+    - description: Identity Store ID.
+      name: IdentityStoreId
+      required: false
+    - description: The name of the group to create.
+      name: displayName
+      required: false
+    - description: The description of the group to create.
+      name: description
+      required: false
+    description: Creates a new IAM Identity Center group for your AWS account.
+    name: aws-iam-identitycenter-create-group
```

Comment:
All of the arguments for this command are optional what will be the behavior if none of the args are used?